lib/modules/filteReddit.js

```diff
@@ -869,15 +869,15 @@ function scanEntries(ele) {
 			// Run async (slow) filters separately
 			// Note: These won't be counted in numFiltered
 			(async () => {
-				filtered = removeVisitedLinksActive && await isURLVisited(postLink);
+				if (postLink) filtered = removeVisitedLinksActive && await isURLVisited(postLink.href);
```

Comment:
Hmmm, now that I look at this a second time I think my concerns were misplaced.

`removeVisitedLinksActive` is always false on pageload, so we'll never hit an `await` during the initial filter, where performance really matters.

So you could put this back to how it was, without the async IIFE.